docs/vocabulary.md

```diff
@@ -75,6 +76,17 @@ Some of the core ArgoCD concept are listed below: [[16]]
 - **Sync phases** [[18]]: Pre-sync, sync and post-sync allow for controlled orchestrated execution of sync operations.
 - **Sync waves**: Within any one sync phase, an additional control of sequential execution is accomplished using waves, to ensure certain resources are healthy before subsequent resources are synced.
 
+### Codefresh
```

Comment:
Same here. Please move Codefresh after CircleCI.

Reply:
Yes, sorry about that